source/puppet/5.3/hiera_config_yaml_5.md

```diff
@@ -344,34 +344,34 @@ hierarchy:
     paths:
       - "%{trusted.certname}.xml"
       - common.xml
-  ```
+```
 
 {:.reference}
 ## Configuring a hierarchy level: general format
 
 Hiera supports custom backends.
 
-See the backend’s documentation for configuring hierarchy levels.
+See the backend's documentation for configuring hierarchy levels.
 
 Each hierarchy level is represented by a hash which needs the following keys:
 
-* `name` — A name for this level, shown in debug messages and `--explain` output.
-* A backend key, which must be one of:
-	* `data_hash`
-	*`lookup_key`
-	*`data_dig` - a more specialized form of `lookup_key`, suitable when the backend is for a database. `data_dig`  resolves dot separated keys, whereas `lookup_key` does not.
```

Comment:
`lookup_key` and `data_dig` didn't render correctly because they didn't have a space between the bullet and the text. Applying consistent four-character indents fixes this.